sir_cake/store/utils.py

```diff
@@ -0,0 +1,70 @@
+from django.contrib import messages
+from django.core.exceptions import ObjectDoesNotExist
+
+from seller.models import Item, Order
+from users.models import AnonymousUser
+from .models import BasketItem
+
+
+def add_one_item_to_basket(request):
+
```

Comment:
Maybe add a short comment at the top of the function that says something like `This function is only called with POST requests`.